uint8 type
uint8 autopilot
uint8 base_mode
uint32 custom_mode
uint8 system_status
uint8 system_id
